flang/lib/Lower/Allocatable.cpp

```diff
@@ -997,12 +997,14 @@ Fortran::lower::genMutableBoxRead(Fortran::lower::FirOpBuilder &builder,
   auto rank = box.rank();
   if (box.isCharacter()) {
     auto len = lengths.empty() ? mlir::Value{} : lengths[0];
-    if (rank)
+    if (rank) {
```

Comment:
Obviously, ignore these changes. I accidentally included this.

Reply:
LLVM style suggests not including curlies.